controllers/jobsController.js

```diff
@@ -0,0 +1,150 @@
+// controllers/jobsController.js
+const Jobs = require("../models/jobs.model");
+const sendResponse = require("../utils/response");
+
+const create = async (req, res) => {
+  try {
+    const { title, business, location, language, level, sectorTags, employmentType, experienceLevel, isRemote } = req.body;
+
+    if (!(title || business || location || language || level || sectorTags || employmentType || experienceLevel)) {
+        sendResponse(res, 400, 'Missing required fields');
+        return;
+    }
+
+    const job = new Jobs({
+        userId: req.userId,
+        title,
+        business,
+        location,
+        language,
+        level,
+        sectorTags,
+        employmentType,
+        experienceLevel,
+        isRemote,
+    });
+    job.save();
```

Comment:
there should be await in this?

Reply:
No need to wait for the document to get saved.

Reply:
why? We need to let user know if it is saved or not no? There could be various reasons for failures like db connectivity, schema validation etc